albinos-editor/src/albinospkg/cli.nim

```diff
@@ -2,24 +2,55 @@ import terminal
 import strutils
 import rdstdin
 import linenoise
+import cmd
+import ../libalbinos/albinos
```

Comment:
Is this acceptable style?

Reply:
Totally I saw it in the official Nim package.